src/common/i18n/translations/type.ts

```diff
@@ -272,6 +272,12 @@ export type Translations = {
       primaryActionText: string;
       secondaryActionText?: string;
     };
+    wrongPaymentReceiptNumber: {
+      title: string;
+      body?: string;
+      primaryActionText: string;
+      secondaryActionText?: string;
```

Comment:
Actually no need add this, since no `secondaryActionText`. I know some of the other objects in here have it but I intended to clear those out